admin/controllers/results.js

```diff
@@ -21,12 +21,16 @@ controller.getViewResultsPage = async (req, res, next) => {
   res.locals.pageTitle = 'Provisional results'
   req.breadcrumbs('Results')
   let pupilResultData
+  let generatedAt
+  let redisResult
   let groups
   let checkWindow
   let isHdfSubmitted
   try {
     checkWindow = await checkWindowV2Service.getActiveCheckWindow()
-    pupilResultData = await resultService.getPupilResultData(req.user.schoolId, checkWindow && checkWindow.id)
+    redisResult = await resultService.getPupilResultData(req.user.schoolId)
```

Comment:
are we agreed that we just show an error page if nothing is in the cache? seems overkill to me.
This is a problem with other areas of the system too - sometimes we know what the problem is likely to be, but instead of showing an explanation, we just show an error code.  I would prefer to show an explanation.

Reply:
thanks for demo, i see it will now show an explanation